app/stack/route.js

```diff
@@ -14,17 +14,17 @@ export default Ember.Route.extend({
     return tokens.join(' - ');
   },
 
+  setupController: function(controller, model) {
+    controller.set('model', model);
+    controller.set('organizations', this.store.find('organization'));
```

Comment:
since `setupController` is a synchronous callback (as opposed to async callbacks like `model`, `afterModel`, `beforeModel`), but `this.store.find('organization')` is an async call, the controller will has `organizations` set to a promise proxy wrapping the in-flight ajax call for organizations. As soon as the API responds the content will be made available to the template but until then the stack template will have an empty space where the organizations would be. And since the stack sidebar relies on the stacks, it may also be empty for a moment for each organization if the `store.find('organizations')` request comes back significantly before the `store.find('stacks')` request does.

This is probably fine as it is, just wanted to clarify that that is what is going on here. It most likely makes the most sense to let this template start rendering before the orgs and stacks have been loaded.

If you wanted to get fancy you could check the `isLoading` property of `organizations` and I believe something like this would work (in the stack/template.hbs):
```
{{#each organizations as |organization|}}
  stack-sidebar etc
{{else if organizations.isLoading}}     {{! the store.find ajax call is in-flight}}
  loading orgs...
{{else}}     {{! not loading anymore and the organizations.length is 0}}
  you have no organizations
{{/each}}
```